basic_system/src/system_functions/point_evaluation/consts.rs

```diff
@@ -53,7 +56,10 @@ pub const G2_BY_TAU_POINT:
         infinity: false,
     };
 
-#[cfg(all(target_arch = "riscv32", feature = "proving"))]
+#[cfg(any(
+    all(target_arch = "riscv32", feature = "proving"),
+    all(feature = "proving", fuzzing)
+))]
```

Comment:
I feel like this is simpler to understand, no? Not sure I added the right amount of closing ) btw
```suggestion
#[cfg(all(feature = "proving", any(target_arch = "riscv32", fuzzing)))]
```